src/pages/ThorChainLP/components/ReusableLpStatus/ReusableLpStatus.tsx

```diff
@@ -100,7 +100,7 @@ export const ReusableLpStatus: React.FC<ReusableLpStatusProps> = ({
 
   const handleComplete = useCallback(
```

Comment:
with this update I would rename this function to `handleStatusUpdate` and propagate accordingly

Reply:
Makes sense 👍